README.md

```diff
@@ -1,135 +1,170 @@
-> Future work will aim to help optimise the hyperparameters of the entire pipeline. 
+🤗 Diarizers is a library for fine-tuning [`pyannote`](https://github.com/pyannote/pyannote-audio/tree/main) speaker 
+diarization models using the Hugging Face ecosystem. It can be used to improve performance on both English and multilingual 
+diarization datasets with simple example scripts, with as little as ten hours of labelled diarization data and just 5 minutes
+of GPU compute time.
 
 ## 📖 Quick Index
 * [Installation](#installation)
 * [Train](#train)
-* [Inference](#inference)
+* [Evaluation](#evaluation)
+* [Inference](#inference-with-pyannote)
 * [Results](#Results)
-* [Adding new datasets](#addingnewdatasets)
+* [Adding new datasets](#adding-new-datasets)
 * [Acknowledgements](#acknowledgements)
 * [Citation](#citation)
 
 ## Installation
 
-Diarizers has light dependencies and can be installed with the following lines: 
+First, clone the repository and install the dependencies:
 
 ```sh
-pip install git+https://github.com/huggingface/diarizers.git
-pip install diarizers[dev]
+git clone https://github.com/huggingface/diarizers.git
+cd diarizers
+pip install -e .
 ```
 
-You'll need to generate a [user access token](https://huggingface.co/docs/hub/en/security-tokens), and login with: 
+To load pre-trained diarization models from the Hub, you'll first need to accept the terms-of-use for the following two models:
+1. [pyannote/segmentation-3.0](https://hf.co/pyannote/segmentation-3.0)
+2. [pyannote/speaker-diarization-3.1](https://hf.co/pyannote/speaker-diarization-3.1)
+
+And subsequently use a Hugging Face authentication token to log in with: 
 
 ```
-pip install -U "huggingface_hub[cli]"
-hugging-cli login
+huggingface-cli login
 ```
 
 ## Train
 
-When fine-tuning a `pyannote` segmentation model on a given dataset, be sure to specify this: 
-
-- `dataset_name`: Specify a dataset from the Hub on which to fine-tune your model.  
-- `dataset_config_name`:  If the dataset contains multiple language subsets, select the language ID of the subset you want to train on.
-- `train_split_name`: Specify which dataset split is to be used for training. Default is 'train'. 
-- `eval_split_name`: Specify which dataset split is to be used for validation. Default is 'validation'. 
-
-If the data set doesn't already contain a train and a validation split, you can automatically split it into train-val-test (90-10-10) using: 
-
-- `do_split_on_subset`: Specify the subset of the dataset you want to split into train-val-set.
+The script [`train_segmentation.py`](train_segmentation.py) can be used to pre-process a diarization dataset and subsequently
+fine-tune the `pyannote` segmentation model. In the following example, we fine-tune the segmentation model on the Japanese
+subset of the [CallHome dataset](https://huggingface.co/datasets/diarizers-community/callhome), a conversational dataset
+between native speakers:
 
-```
-python3 train_segmentation.py
```

Comment:
Please make sure to actually run these examples before sharing with others - the one here didn't work for number of syntax reasons